package.json

```diff
@@ -66,11 +67,12 @@
   },
   "dependencies": {
     "@emotion/css": "^11.10.6",
-    "@grafana/data": "^10.4.1",
```

Comment:
Just FYI, because I recently wanted to upgrade.

```
 @grafana/scenes@4.5.5" has incorrect peer dependency "@grafana/data@^10.0.3".
warning " > @grafana/scenes@4.5.5" has incorrect peer dependency "@grafana/runtime@^10.0.3".
warning " > @grafana/scenes@4.5.5" has unmet peer dependency "@grafana/schema@^10.0.3".
warning " > @grafana/scenes@4.5.5" has incorrect peer dependency "@grafana/ui@^10.0.3".

```

Reply:
Yeah we need newer grafana for handling the links within the viz on click. Hopefully this doesn't cause any trouble, and if so we can revert back to older grafana (or try to update scenes) in another PR.